src/app/screens/book-list/book-list.component.ts

```diff
@@ -9,12 +12,21 @@ import { BookService } from '@app/services/book.service';
 export class BookListComponent implements OnInit {
   books;
   filterValue = '';
+  shoppingBooks;
 
-  constructor(private bookService: BookService) { }
+  constructor(private bookService: BookService, private shoppingService: ShoppingCartService) { }
 
   ngOnInit(): void {
-    this.bookService.getBooks()
-      .subscribe(({page}) => this.books = page)
+    forkJoin(
+      this.bookService.getBooks()
+        .pipe(tap(({page}) => this.books = page)),
+      this.shoppingService.shoppingBooks
+        .pipe(tap(books => this.shoppingBooks = books))
+    ).subscribe()
```

Comment:
Nooooope. :sad-harold:

```suggestion
    forkJoin({
      books: this.bookService.getBooks(),
      page: this.shoppingService.shoppingBooks
     }).subscribe(({ books, page })=>{
        this.shoppingBooks = books
        this.books = page
     })
```